tox.ini

```diff
@@ -18,41 +23,57 @@ commands = pre-commit run --all-files --show-diff-on-failure --color=always
 [testenv:py{,38,39,310,311,312}{,-fast}]
 depends: pre-commit
 use_develop = fast: True
-skip_sdist = fast: True
 # Needed on Windows to test data directory creation
 passenv = LOCALAPPDATA
+setenv = COVERAGE_FILE = {env:COVERAGE_FILE:.coverage}
 extras = dev
 # 2023-04-22 see pkgenv ↑
 download = {[pkgenv]download}
 commands =
-    !fast : python -m coverage run -m pytest {posargs:-vv}
-    fast : python -m pytest {posargs:-vv -n auto}
+    !fast : python -m coverage run -m pytest {posargs:-vv --color yes}
+    fast : python -m pytest {posargs:-vv --color yes -n auto}
```

Comment:
funny story if you miss it in my commit...

Upon realizing that `pytest` wasn't running in color after adding `FORCE_COLOR`, it dawned on me that `tox` was masking the env var for `pytest`. However, when I passed `FORCE_COLOR=1` to `pytest`, a lot of the tests failed. This is because we're using `capsys` to capture the output....which becomes full of ANSI control chars.

This turns out to have been a problem before `FORCE_COLOR` was added to CI....but fyi to keep in mind.